packages/benchmark/run-tests.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+set -e 
+
+echo $(yarn --version)
+
+are_engines_installed() {
+    [ -d "$HOME/.esvu/engines/xs" ] && [ -d "$HOME/.esvu/engines/v8" ]
+}
+
+if are_engines_installed; then
+    echo "Engines already installed. Skipping installation."
+else
+    echo "Installing engines..."
+    INSTALL_OUTPU_XS=$(yarn dlx esvu install xs 2>&1) || INSTALL_STATUS_XS=$?
+    INSTALL_OUTPU_V8=$(yarn dlx esvu install v8 2>&1) || INSTALL_STATUS_V8=$?
+fi
+
+if [ -n "$INSTALL_STATUS_XS" ] || [ -n "$INSTALL_STATUS_V8" ]; then 
+    if are_engines_installed; then
+        echo "Engines installed successfully despite esvu error."
+    else
+        echo "Error installing XS or V8:"
+        echo "$INSTALL_OUTPU_XS"
+        echo "$INSTALL_OUTPU_V8"
+        exit 1
+    fi
+fi
```

Comment:
I'm not comfortable with `yarn test` modifying $HOME.
```suggestion
if are_engines_installed; then
    echo "Engines already installed. Skipping installation."
else
    echo "xs and/or v8 not found in $HOME/.esvu/engines; please esvu install them."
    # The bash "command not found" exit status code is 127.
    # https://www.gnu.org/software/bash/manual/html_node/Exit-Status.html#Exit-Status-1
    exit 127
fi
```